config/application.rb

```diff
@@ -33,6 +33,9 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
 
+    # Add app/validators to autoload paths
+    config.autoload_paths << Rails.root.join( 'app', 'validators' )
```

Comment:
Subdirectories of `app/` shouldn't need to be added to `config.autoload_paths`, line 37 can be removed.

> Autoload paths automatically pick any custom directories under app. For example, if your application has app/presenters, or app/services, etc., they are added to autoload paths. https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#autoload-paths

Reply:
Did not work on my dev environment (worked fine in tests and CI mind you). Have I broken my local setup somehow?

Reply:
It might have been that any rails servers and spring processes needed to be restarted after the `app/validators/` directory was added. Perhaps try these steps in order:

1. removing the `config.autoload_paths << ...` line
2. stopping any running rails servers
3. Run `bin/spring stop` to stop any spring processes
4. start a rails server `bin/rails s`
5. check the safe slug validator still works